raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,374 @@
ce or information to the public or open to public inspection; any documentation prepared, owned, used or retained by any ministry or agency in pursuance of law or in connection with the transaction of public business. "Published notice" means notice that shall be placed in at least one daily newspaper of major national circulation, and/or one newspaper having a district circulation, and shall be broadcast on a popular local station in English and at least one vernacular language relevant to the venue; and shall be disseminated as widely as is practicable by Environmental County officers through the county and district environmental committees, NGOs and CBOs. "Radiation" includes ionizing radiation and any other radiation likely to have adverse effects on human health and the environment. "Regulations" means rules and regulations made under this Law. "scoping" means an early and open process for determining the scope of the Issues to be addressed and for identifying the significant issues related to the proposed activity. "segment" in relation to the environment means any portion or portions of the environment expressed in terms of volume, space, areas, quantity, quality or time or any combination thereof. "ship" includes every description of vessel or craft or floating structure. "soil" means an upper layer of earth and includes sand, rock, shells, minerals, vegetation and soil flora, and fauna in the soil and derivatives such as dust. "Standard" means the limits of pollution established under this Act or under the regulations made under this Act or any other law. "Sustainable development" means development that meets the needs of the present generation without compromising the ability of future generations to meet their needs by maintaining the carrying capacity of the supporting ecosystems. "Sustainable use" means present use of the environment or natural resources which does not compromise or impose on the ability to use the same by future generations or degraded the carrying capacity of supporting ecosystems. "Territorial water" means territorial waters off Liberia's coastline. Trade" means any trade, business or undertaking whether originally carried on at fixed premises or at varying places which may result in the discharge of substances and energy and includes any activity prescribed to be a trade, business or undertaking for the purposes of this Law. "Waste" means any substance that may be prescribed as waste or any matter, whether liquid, solid, gaseous, or radioactive, which is discharged, emitted or deposited in the environment. "Water" means drinking water, water used for domestic and industrial purposes and for animals, plants, and includes sea, river, stream, water-course, reservoir, well, dam, canal, channel, lake, swamp, open drain or underground water. "Wetland" means an area permanently or seasonally flooded or saturated by water where plants and animals have become adapted. "Wise use" means sustainable use of natural resources.
```

Comment:
I missed this previously, but please replace `(3) Agency"` with `(3) "Agency"`. Also, please replace `coastline. Trade"` with `coastline. "Trade"`.